sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/StorageTest.java

```diff
@@ -136,6 +156,28 @@ public void disablePersisted() {
         verify(channel, never()).addListener(any(Channel.Listener.class));
     }
 
+    @Test
+    public void listEndToEnd() throws Exception {
+        HttpClientRetryer httpClient = mock(HttpClientRetryer.class);
+        whenNew(HttpClientRetryer.class).withAnyArguments().thenReturn(httpClient);
+        when(SharedPreferencesManager.getBoolean(STORAGE_ENABLED_KEY, true)).thenReturn(false);
+        Storage storage = Storage.getInstance();
+        Channel channel = start(storage);
+
+        AppCenterFuture<Documents<TestDocument>> docs = Storage.list(PARTITION, TestDocument.class);
+    }
+
+    @Test
+    public void replaceEndToEnd() throws Exception {
+        HttpClientRetryer httpClient = mock(HttpClientRetryer.class);
+        whenNew(HttpClientRetryer.class).withAnyArguments().thenReturn(httpClient);
+        when(SharedPreferencesManager.getBoolean(STORAGE_ENABLED_KEY, true)).thenReturn(false);
+        Storage storage = Storage.getInstance();
+        Channel channel = start(storage);
+
+        AppCenterFuture<Document<TestDocument>> doc = Storage.replace(PARTITION, DOCUMENT_ID, new TestDocument(TEST_FIELD_VALUE));
+    }
+
     @Test
     public void readEndToEnd() throws Exception {
         HttpClientRetryer httpClient = mock(HttpClientRetryer.class);
```

Comment:
add comments /* When */ /* Call */ /* verify */

Reply:
These particular ones are for code coverage, basically, stubs